### PR TITLE
Change the output format to be very similar to a TESS FFI

### PR DESCRIPTION
**Pull request by amcody**
K2mosaic now creates a multi-extension file instead of a single extension image. This should be compatible with TESS FFI format, as well as include the key K2 header keywords. It also allows the user a new option of selecting whether they want background subtracted output images (if available). 

To do:
-Figure out why this is so slow!
-Need to pass time keywords to the headers (how?)
-Currently many header keywords are hard coded in. Better to move to a template file?
-Complete image and uncertainty extensions headers
-Decide what to do about WCS, which currently relies on a K2 FFI being stored (AMC doesn't like)

@barentsen when would be a good time to discuss?

### CODE REVIEW

File: k2mosaic/mosaic.py [hunk 1]
```diff
@@ -71,34 +80,151 @@ def add_wcs(self):
...
+        hdu.header.cards['NEXTEND'].comment = 'number of standard extensions'
+        hdu.header['EXTVER'] = 1
+        hdu.header.cards['EXTVER'].comment = 'extension version number (not format version)'
+        hdu.header['SIMDATA'] = False
+        hdu.header['ORIGIN'] = "NASA/Ames"
+        hdu.header['DATE'] = datetime.datetime.now().strftime("%Y-%m-%d")
+
+        ### Put tstart/stop and date-obs/end in here! How do we pull in time, and where do we store it??
+        if self.time is not None:
+            pass  # compute the time values
```
Comment (barentsen) [inline k2mosaic/mosaic.py]: This will need to be implemented.

**Comment by amcody** [comment]
This pull request is now ready for review, @barentsen! I sent you a sample .fits file on slack- see if you can find any errors in the headers.

**Comment by amcody** [comment]
I fixed a small bug... now let's see if Travis coughs..

**Comment by barentsen** [comment]
Sanity check of the times:

Header of `https://www.dropbox.com/s/mj1dfk0144raaw0/k2mosaic-c91-ch31-cad125245_wcs.fits` says:
```
DATE-OBS= '2016-04-22T15:02:53.276791Z' / TSTART as UTC calendar date           
DATE-END= '2016-04-22T15:32:18.739675Z' / TSTOP as UTC calendar date
```

Original TPF says:
```python
> KeplerTargetPixelFile('ref/ktwo200068780-c91_lpd-targ.fits.gz')[2].astropy_time.iso
array(['2016-04-22 15:17:46.080'], dtype='<U23')
```

Very close, though I don't understand why the latter is not exactly in the middle of DATE-OBS and DATE-END.

**Comment by barentsen** [comment]
The comment above is invalid because I was using a TPF from a different channel.  Using a TPF from the same channel yields the expected times!

**Comment by barentsen** [comment]
This looks great!  Thanks for working on this @amcody.  Merging in 3 ... 2 ...

**Pull request merged**